scripts/ci/github_actions/driver/wait_for_driver_to_be_ready.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
```

Comment:
if `wait_for_k8s_ready.sh` doesn't have `_to_be`, then this filename shouldn't have it either